lib/geocoder/virtualearth.js

```diff
@@ -63,7 +63,7 @@ VirtualEarthGeocoder.prototype._reverse = function(value, callback) {
     key: this.options.apiKey
   };
 
-  var endpoint = this._endpoint + "/" + value.lat + "," + value.lon;
+  var endpoint = this._endpoint + '/' + value.lat + ',' + value.lon;
```

Comment:
Would this be better as `${this._endpoint}/${val.lat},${val.lon}`;?

Reply:
If I can find somewhere else in the project that uses template literals, then yes.

Reply:
I could not find any, and since it [says](https://github.com/nchaulet/node-geocoder/blob/master/CONTRIBUTING.md) "Follow existing coding style", I think we should leave it.